Polyhedron/demo/Polyhedron/Scene_surface_mesh_item.cpp

```diff
@@ -87,6 +87,7 @@ struct Scene_surface_mesh_item_priv{
   mutable bool is_filled;
   mutable bool isinit;
   mutable std::vector<unsigned int> idx_data_;
+  mutable std::map<unsigned int, unsigned int> current_indices; //map im values to ghosts-free values
```

Comment:
Seems to be used only by `compute_elements()`. Why does it need to be a member of the item_priv object?